src/main/java/com/cincialert/dao/IUserDAO.java

```diff
@@ -0,0 +1,18 @@
+package com.cincialert.dao;
```

Comment:
Project was missing DAO object for your dto. You need the dao as it is the layer that is supposed to access your data.